ipwb/util.py

```diff
@@ -39,16 +45,26 @@
 dtPattern = re.compile(r"^(\d{4})(\d{2})?(\d{2})?(\d{2})?(\d{2})?(\d{2})?$")
 
 
-def isDaemonAlive(hostAndPort="{0}:{1}".format(IPFSAPI_HOST, IPFSAPI_PORT)):
+def createIPFSClient():
+    try:
+        return ipfsapi.Client(IPFSAPI_MUTLIADDRESS)
+    except ipfsapi.exceptions.AddressError:
+        return None  # Malformed multiaddress for the daemon
+
+
+def isDaemonAlive(daemonMultiaddr=IPFSAPI_MUTLIADDRESS):
     """Ensure that the IPFS daemon is running via HTTP before proceeding"""
-    client = ipfsapi.Client(f"/dns/{IPFSAPI_HOST}/tcp/{IPFSAPI_PORT}/http")
+    client = ipwbUtils.createIPFSClient()
```

Comment:
```suggestion
    client = createIPFSClient()
```

It is a local function.

Reply:
Yeah, noticed that after I pushed. Fixed in 05e7eba.